bastion_current_ip.py

```diff
@@ -0,0 +1,76 @@
+# obtain current public ip, create a variables.tfvars file 
```

Comment:
Run this through a linter, so your code is consistant with PEP8 standard.  I suggest pylint.